selfhost/typechecker.jakt

```diff
@@ -5469,7 +5469,7 @@ struct Typechecker {
                 }
             }
             Struct(lhs_struct_id) => {
-                if lhs_type_id.equals(rhs_type_id) or .is_subclass_of(ancestor_type_id: rhs_type_id, child_type_id: lhs_type_id) {
+                if lhs_type_id.equals(rhs_type_id) {
```

Comment:
don't we still need this?
it should just be `is_subclass_of(lhs, rhs)` instead?

Reply:
The version with (lhs, rhs) is already there, just little below

Reply:
Ah I see